.env.production

```diff
@@ -1,2 +1,3 @@
 REACT_APP_GOOGLE_ANALYTICS_MEASUREMENT_ID="G-8JBQYLS8HB"
 REACT_APP_NETWORK_URL="https://mainnet.infura.io/v3/28e6f2ed0e5747e6bef772f27372f62d"
+REACT_APP_ALTERNATIVE_NETWORK_URL = "https://mainnet.infura.io/v3/28e6f2ed0e5747e6bef772f27372f62d"
```

Comment:
I added this value because of the local host test.
On local host we are using `http://127.0.0.1:8545` as `REACT_APP_NETWORK_URL`, and it don't create default network provider if there is no metamask.

But I believe this issue is only required on `production` and so we don't need this variable.

Reply:
@nkrulikovsky we don't need this anymore right? Please remove if so

Reply:
I already removed )